runtime/wasm/Cargo.toml

```
@@ -13,29 +13,29 @@ polkadot-primitives = { path = "../../primitives", default-features = false }
 safe-mix = { version = "1.0", default-features = false }
 parity-codec = { version = "2.1", default-features = false }
```

why keep the old version here?
```suggestion
 parity-codec = { version = "3.0", default-features = false }
```